libethereum/Executive.cpp

```diff
@@ -331,11 +354,6 @@ bool Executive::create(Address _sender, u256 _endowment, u256 _gasPrice, u256 _g
 	// Schedule _init execution if not empty.
 	if (!_init.empty())
 		m_ext = make_shared<ExtVM>(m_s, m_envInfo, m_sealEngine, m_newAddress, _sender, _origin, _endowment, _gasPrice, bytesConstRef(), _init, sha3(_init), m_depth);
-	else if (m_s.addressHasCode(m_newAddress))
```

Comment:
Why do you remove this? I guess this can break a bunch of tests that check consensus around address collision. 
This is the branch for empty init code, so it's not contract creation and I think this should work as earlier.